.gitlab-ci.yml

```diff
@@ -1,68 +1,157 @@
+
 stages:
-  - generator
-  - test-basic
-  - test-required
-  - documentation
-  - test-additional-v1.5
-  - test-additional-v1.6
+  - test # this checks the viability of the code
+  - docdata # this builds stuff and assets for the documentation
```

Comment:
`docdata` - I'd suggest `documentation-assets`

Reply:
`stuff` is sort of not very precise... ? `assets` should be enough (in the comment)

Reply:
can it actually have a t-a-g in a name? (I'll try)